src/plenigo/models/ProductBase.php

```diff
@@ -175,6 +181,21 @@ public function setCustomAmount($condition)
         $this->customAmount = safe_boolval($condition);
     }
 
+    public function isSubscriptionRenewal()
+    {
+        return $this->subscriptionRenewal;
+    }
+
+    public function getSubscriptionRenewal()
```

Comment:
There is no need for 2 getters, you can leave the isSubscriptionRenewal and remove the getter.